src/views/resultsView/createScanProvider.ts

```diff
@@ -153,8 +154,12 @@ export async function createScan(
   }
 
   updateStatusBarItem(constants.scanCreatePreparing, true, statusBarItem);
-  await createScanForProject(context, logs);
-
+  try {
+    await createScanForProject(context, logs);
+  } catch (error) {
+      setScanButtonDefaultIfScanIsNotRunning(context);
```

Comment:
can we add a test to this flow? (UT is enough)

Reply:
We didn't have a UT.